cognite/neat/rules/models/dms/_schema.py

```diff
@@ -203,6 +216,21 @@ def from_data_model(
             reference=ref_schema,
         )
 
+    @classmethod
+    def _connection_references(cls, view: dm.View) -> set[dm.ViewId]:
```

Comment:
We should raise warning that our data model is not complete since it is pointing to types outside of the data model.